code/modules/mechs/components/body.dm

```diff
@@ -225,6 +205,9 @@
 	max_damage = 150
 	power_use = 0
 	climb_time = 7 // Quickest entry because it's unsealed
+	armor = list(melee = 24, bullet = 10, energy = 4, bomb = 60, bio = 100, rad = 0)
+	shielding = 5
+	rear_mult = 1.1
```

Comment:
Unusual? Hints towards this stat outlier should be added to the description of the item.